systemtest/src/test/java/io/strimzi/systemtest/SecurityST.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2018, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.systemtest;
+
+import io.strimzi.test.ClusterOperator;
+import io.strimzi.test.JUnitGroup;
+import io.strimzi.test.KafkaFromClasspathYaml;
+import io.strimzi.test.Namespace;
+import io.strimzi.test.StrimziRunner;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.util.stream.IntStream;
+
+import static org.hamcrest.Matchers.containsString;
+
+@RunWith(StrimziRunner.class)
+@Namespace(SecurityST.NAMESPACE)
+@ClusterOperator
+public class SecurityST extends AbstractST {
+
+    private static final Logger LOGGER = LogManager.getLogger(SecurityST.class);
+    public static final String NAMESPACE = "security-cluster-test";
+
+    @Test
+    @JUnitGroup(name = "regression")
+    @KafkaFromClasspathYaml()
+    public void testCertificates() {
+        LOGGER.info("Running testCertificates {}", CLUSTER_NAME);
+
+        String kafkaCertificateChain = "s:/O=io.strimzi/CN=my-cluster-kafka\n" +
```

Comment:
If we are already testing this, we should also test the new SAN values added by @ppatierno in one of the other PRs.

Reply:
Yeah the #663 is now merged.